prisma-fmt-wasm/scripts/update-fmt-wasm.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
```suggestion
set -euxo pipefail
```
So we get the actual command ran in the logs?

Reply:
If this is needed, I can re-add it, it just felt like un-necessary spam/clutter when running it locally, especially given how small the script is and that it's already adding print info as to where it is

Reply:
It's indeed quite verbose with -x

Reply:
since it's something we only run locally maybe it's just fine if we minimize the output for now and see how it goes, we can always change this